_about/projects/wai-coop/symposium2.md

```diff
@@ -26,115 +67,848 @@ github:
+
+JUTTA TREVIRANUS: Right. That brings in the issue of classification and labeling, and the need to box or classify specific things. And many of these things are very fuzzy context, and classifiers are also determined hierarchically and maybe there is...
+
+SHARI TREWIN: Yes. Maybe we don't need a perfect classifier, but we need a good dialogue where the system knows what questions to ask to help the person decide.
+
+JUTTA TREVIRANUS: Right. And, oh, I just saw a message from Carlos saying we are only down to a few more minutes. Can we fit in one more question?
+
+SHARI TREWIN: I actually have to stop at the top of the hour.
+
+JUTTA TREVIRANUS: Oh, okay. We will have an opportunity to answer the questions that people have submitted in the question and answer dialogue, and we have access to those, so Shari will be able to respond to some of these additional questions that have been asked. Apologies that we went a little over time, Carlos. I will turn it back over to you.
+
+CARLOS DUARTE: No. Thank you so much. And thank you, Shari, for the keynote presentation. Thank you Shari and Jutta, I was loving this discussion. It is really unfortunate that we have to stop now. But, thank you so much for your presentations. Thank you, also, to all the panelists yesterday and today for making this a great symposium. Lots of interesting and thought provoking ideas.
+
+And, thank you all for attending. We are at the top of the hour, so we are going to have to close. Let me just, a final ask from me. When you exit this Zoom meeting, you will receive a request for completing a survey, so if you can take a couple of minutes from your time to complete it, it will be important information for us to make these kinds of events better in the future.
+
+Okay. Thank you so much, and see you in the next opportunity.
+
+
+{% include excol.html type="end" %}
+
+
+## Session Videos
+
+{% include video-playlist.html%}
+
+## Symposium Full Transcripts
+{:#transcripts}
+
+- [Transcriptions of the first day](../symposium2_day1_transcript.html)
+- [Transcriptions of the second day](../symposium2_day2_transcript.html)
+
+
+## Conclusions and Future Directions
```

Comment:
I wonder if this section should go before the vidoes? (Or, if we integrate the videos with above (which Kevin will follow up on), then this comment is moot. :-)

Reply:
@shawna-slh Do you mean the "Conclusions and Future Directions" section?